packages/@uppy/transloadit/src/Assembly.ts

```diff
@@ -263,13 +286,15 @@ class TransloaditAssembly extends Emitter {
   /**
    * Stop updating this assembly.
    */
-  close () {
+  close(): void {
     this.closed = true
     if (this.#sse) {
       this.#sse.close()
       this.#sse = null
     }
-    clearInterval(this.pollInterval)
+    if (this.pollInterval) {
+      clearInterval(this.pollInterval)
+    }
```

Comment:
There's actually no issue with calling `clearInterval` with a nullish value, it's a nop-op guaranteed by spec.
```suggestion
    clearInterval(this.pollInterval!)
```